packages/hash/frontend/src/components/RemoteBlock/blockRenderer.tsx

```diff
@@ -18,7 +20,6 @@ export const BlockRenderer: FunctionComponent<BlockRendererProps> = ({
   properties,
   sourceUrl,
 }) => {
-  // @todo remove this React default when we update all blocks to 0.2
```

Comment:
Why's this comment removed?

Reply:
Some blocks like the chart block remain pre-0.2 (although this one at least is presently non-functional in the HASH app anyway)